### ISSUE TITLE
[Bug] VMulti | Win Portrait mode

### ISSUE THREAD

**Issue by C0D3-M4513R** [X9VoiD/VoiDPlugins#20]
## Description

When putting your monitor in portrait mode, the VMulti Absolute mode breaks!
eg. 1080x1920 instead of 1920x1080
## Steps to reproduce
- Put your monitor in portrait mode
- Use VMulti Absolute Mode
- Try to make your tablet actually fill the whole screen
## System Information:

| Name             | Value |
| ---------------- | ----- |
| Operating System | Win 10
| Software Version | v.0.5.3.1
| Tablet           | MTE-450

EDIT: Now I've noticed, that after switching from vertical to horizontal, that the problem persists.
EDIT2: It persists now even across reboots? WTH? install problem? gotta try and reinstall the driver/plugin!

**Comment by X9VoiD** [comment]
Restart OTD, then reconfigure your screen area after switching to portrait mode. OTD doesn't support detection of screen state changes (resolution, orientation). Will close since this is not a problem of the plugin itself.

**Comment by C0D3-M4513R** [comment]
Ok, so that the Driver just doesn't move the cursor over the 1st half of the configured display width, after one has rebooted/upgraded the pc is perfectly normal?
This is the behavior, that I have been seeing.
I have rebooted my pc, and I have changed the OTD settings before opening this issue.

This Layout works, to bind my tablet to my primary monitor.
![image](https://user-images.githubusercontent.com/28912031/123091178-5208fb00-d429-11eb-8cbb-39e3cf4e78dc.png)
This Layout doesn't work. It binds the Tablet to 1/2 of the primary monitor.
![image](https://user-images.githubusercontent.com/28912031/123091247-68af5200-d429-11eb-9b51-df0d1c4c5556.png)
